app/build.gradle

```diff
@@ -7,20 +7,40 @@ apply plugin: 'kotlin-android-extensions'
 android {
     compileSdkVersion 29
     buildToolsVersion "29.0.2"
-    defaultConfig {
+    defaultConfig
+    {
         applicationId "com.shuttl.packagetest"
         minSdkVersion 16
         targetSdkVersion 29
         versionCode 1
         versionName "1.0"
         testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
+
+        testOptions {
+            execution 'ANDROIDX_TEST_ORCHESTRATOR'
+        }
+        testInstrumentationRunnerArguments clearPackageData: 'true'
     }
     buildTypes {
         release {
             minifyEnabled false
             proguardFiles getDefaultProguardFile('proguard-android-optimize.txt'), 'proguard-rules.pro'
         }
+
+        qa {
```

Comment:
Let's rename this for dev as its for devs primarily.

Reply:
Done